.github/workflows/pr-requirements-check.yml

```
@@ -0,0 +1,58 @@
+# From: https://levelup.gitconnected.com/enforcing-jira-ticket-formatting-in-github-pr-titles-and-commit-messages-78d9755568df
+name: "PR Requirements Check"
+permissions:
+  contents: read
+on:
+  pull_request:
+    branches: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)